sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/signaling/message/Catalog.kt

```diff
@@ -111,15 +111,31 @@ data class RedirectMessage(
 data class OfferMessage(
     @SerializedName("type") val type: String = "offer",
     @SerializedName("sdp") val sdp: String,
+    @SerializedName("version") val version: String? = null,
+
+    @SerializedName("multistream") val multistream: Boolean? = null,
+    @SerializedName("simulcast") val simulcast: Boolean,
```

Comment:
元の内容から変わっているように見えますが意図的ですか？
```suggestion
    @SerializedName("simulcast") val simulcast: Boolean = false,
```

Reply:
ごめんなさいミスです。サジェスト内容を commit します...!